spec/serializers/rest/admin/account_serializer_spec.rb

```diff
@@ -5,6 +5,14 @@
 RSpec.describe REST::Admin::AccountSerializer do
   subject { serialized_record_json(record, described_class) }
 
+  context 'when created_at is populated' do
+    let(:record) { Fabricate :account, user: Fabricate(:user) }
```

Comment:
I havent looked at this one specifically to see if this is true or not - but we should prefer where possible to use the `Fabricate.build ...` form of these, which creates a valid instance of the model class, but does not persist to DB. I suspect for most of these serializers we should be able to just create a normal ruby object to pass in and shouldn't need actual DB records. In cases we do, fine to do so for now but might be interesting followup to see if we can make the serializers rely on passed in objects only and not need to query out while building.

Again I did not look at this one specifically, just general observation.

Reply:
I'm just following the existing code style (e.g., the usage in `status_serializer_spec.rb`, which looks like:

```ruby
  let(:current_user) { Fabricate(:user) }
  let(:alice) { Fabricate(:account, username: 'alice') }
  let(:bob)   { Fabricate(:account, username: 'bob', domain: 'other.com') }
  let(:status) { Fabricate(:status, account: alice) }
```